### PR TITLE
Bump connect to 2021.12.0

### PR DESCRIPTION
**Pull request by scottmmjackson**
- Bump connect version to 2021.12.0

### CODE REVIEW

File: connect/Dockerfile [hunk 1]
```diff
@@ -100,7 +100,7 @@ COPY startup.sh /usr/local/bin/startup.sh
 RUN chmod +x /usr/local/bin/startup.sh
 
 # Download RStudio Connect -----------------------------------------------------#
-ARG RSC_VERSION=2021.11.1
+ARG RSC_VERSION=2021.12.0
```
Comment (aronatkins) [inline connect/Dockerfile:103]: There are additional references. As found by `git grep`:

```
Makefile:8:RSC_VERSION ?= 2021.11.1
connect-content-init/Dockerfile:12:ARG RSC_VERSION=2021.11.1
connect/.env:1:RSC_VERSION=2021.11.1
connect/Dockerfile:103:ARG RSC_VERSION=2021.11.1
docker-compose.yml:27:        RSC_VERSION: 2021.11.1
docker-compose.yml:28:    image: rstudio/rstudio-connect:2021.11.1
```

**Review by aronatkins — CHANGES_REQUESTED** [review]

**Comment by colearendt** [comment]
Docs on an easy way to update: https://github.com/rstudio/rstudio-docker-products/blob/main/RELEASE.md#updating-product-versions

**Comment by scottmmjackson** [comment]
@colearendt not sure that did what we wanted:

```
 M Makefile
 M connect-content-init/Dockerfile
 M connect/.env
 M docker-compose.yml
?? "connect-content-init/Dockerfile ''"
?? "connect/.env ''"
?? "connect/Dockerfile ''"
?? "docker-compose.yml ''"
?? "package-manager/.env ''"
?? "package-manager/Dockerfile ''"
?? "package-manager/rstudio-pm.gcfg ''"
?? "r-session-complete/bionic/.env ''"
?? "r-session-complete/bionic/Dockerfile ''"
?? "r-session-complete/centos7/.env ''"
?? "r-session-complete/centos7/Dockerfile ''"
?? "workbench/.env ''"
?? "workbench/Dockerfile ''"
```

**Comment by scottmmjackson** [comment]
CLA assistant borked

can't agree to the CLA :(

**Comment by scottmmjackson** [comment]
@aronatkins I think we're good for re-review here.

**Comment by colearendt** [comment]
Weird! Sorry for the trouble - I'm curious why that failed - it always works well for me 😅 . @scottmmjackson are you on linux?

**Comment by scottmmjackson** [comment]
@colearendt nope- mac OS here.

**Comment by colearendt** [comment]
🤔  Very strange. I suspect we will need to do some more poking to see what the Makefile was doing there.

**Pull request merged**